docs/source/sql/ml.rst

```diff
@@ -104,11 +97,10 @@ prediction, depending if your model can cope with
 dask dataframes.
 
     * if you are training on relatively small amounts
-      of data but predicting on large data samples
-      (and you are not using a model build for usage with dask
-      from the dask-ml package), you might want to set
-      ``wrap_predict`` to True. With this option,
-      model interference will be parallelized/distributed.
+      of data but predicting on large data samples,
+      you might want to set ``wrap_predict`` to True.
+      With this option, model interference will be
+      parallelized/distributed.
     * If you are training on large amounts of data,
       you can try setting wrap_fit to True. This will
       do the same on the training step, but works only on
```

Comment:
Can't leave a review comment on it but looks like there's still a mention of dask-ml's `ParallelPostFit` under the `SELECT FROM PREDICT` subsection

Reply:
Thanks, if it's ok I can remove this in #866, since the comment corresponds to an identical comment in `predict.py` (and I have no `predict.py` changes in this PR).

Reply:
Sure, that's fine by me 🙂